### PR TITLE
Fix/cannot use enter key to log in

### LINKED ISSUE #1118: Use the Enter Key to sign in
### Overview
Currently, if you click enter after typing in your email, it does not sign you in, you have to click the 'LOGIN' button

### Action Items
- [ ] figure out what happens when the button is clicked
- [ ] figure out what happens when the enter is pushed
- [ ] fix the logic for the enter
- [ ] maybe add some regex validation if there is not one

### PR DESCRIPTION
**Pull request by chukalicious**
Resolves #1118 

Created `useEffect` function to handle a keydown event for the 'enter' key that triggers the `handleLogin()` function created for the component, allowing the user to reach the Success page by pressing the 'enter' key, as well as the form's submit button.

### CODE REVIEW

File: client/src/components/auth/Auth.js [hunk 1]
```diff
@@ -73,7 +77,9 @@ const Auth = () => {
 
   function handleInputChange(e) {
     const inputValue = e.currentTarget.value.toString();
+    validateEmail();
     if (!inputValue) {
+      console.log('inputValue inside the if statement: ', inputValue);
```
Comment (evanyang1) [inline client/src/components/auth/Auth.js:82]: If I recall correctly console logs shouldn't be in the final code

Comment (evanyang1) [inline client/src/components/auth/Auth.js]: If I recall correctly we don't want console logs in the codebase.

```suggestion
      
```

**Review by evanyang1 — APPROVED** [review]
LGTM! Do want someone else to check it first before merging

File: client/src/components/auth/Auth.js (hunk 1 above)
Comment (chukalicious) [reply client/src/components/auth/Auth.js]: Thanks Evan! I didn't see that console.log there. I took it out.

**Pull request merged**